core/src/main/java/org/apache/accumulo/core/client/admin/ImportConfiguration.java

```diff
@@ -73,6 +80,14 @@ interface Builder {
      */
     Builder setKeepMappings(boolean keepMappings);
 
+    /**
+     * set the initial tablet availability for the table
```

Comment:
Could document the behavior when this function is not called, specify what will happen in this case.